tools/nuget/generate_nuspec_for_native_nuget.py

```diff
@@ -389,23 +414,31 @@ def generate_files(line_list, args):
     runtimes = f'{runtimes_target}{args.target_architecture}\\{runtimes_native_folder}"'
 
     # Process headers
+    build_dir = "buildTransitive" if "Microsoft.ML.OnnxRuntime.Gpu" in args.package_name else "build"
```

Comment:
Thanks reminder.
No, it doesn't work with build.py now. 
Because I didn't want to make OnnxRuntime.CSharp.proj too complex.
I thought it'd better that it generates one nuget package (except onnxruntime.managed) for OnnxRuntime.CSharp.proj.

Reply:
We do have folks that use build.py with `--build_nuget` to test. 

Do we maybe need a setting saying whether to split up the GPU package or not so that for local testing it builds a single package?

Reply:
I've moved sub packages generation into csproj file, thus, build.py and CI both work.